app/routes/bdb/utils.js

```diff
@@ -155,6 +158,7 @@ export const getContactedStatuses = (
     ) {
       contacted.splice(contacted.indexOf(status), 1);
     }
+    return null;
```

Comment:
Is there a reason to have a `map` and not a `forEach` here?

Reply:
Yeah, I agree. Moved these to forEach.